presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java

```diff
@@ -591,14 +591,14 @@ public void testDateFormat()
         assertFunction("date_format(" + dateTimeLiteral + ", '%4')", "4");
         assertFunction("date_format(" + dateTimeLiteral + ", '%x %v')", "2001 02");
 
-        String wierdDateTimeLiteral = "TIMESTAMP '2001-01-09 13:04:05.321 +07:09'";
+        String wierdDateTimeLiteral = "TIMESTAMP '2001-01-09 13:04:05.32 +07:09'";
```

Comment:
`321` is easier to get right than `32`. You can't get `32` right unless you are counting from left to right. But you can luckily get `321` right if you are parsing it as millisecond.